micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/QuickStart/QuickSetup.js

```
@@ -28,15 +60,13 @@ const CardC = ({ type, title, content, actions, style }) => {
 
-const FAQ = ({ key, title, content, faqs }) => {
+const FAQ = ({ key, title, content, faqs ,type}) => {
   const { t } = useTranslation();
   return (
     <div style={{ width: "100%" }}>
-      <CardSectionHeader>{t(title)}</CardSectionHeader>
-      <CardSectionSubText style={{ marginTop: "1rem" }}>{t(content)}</CardSectionSubText>
       <div style={{ width: "100%" }}>
         {faqs.map((faq, i) => (
-          <FaqComponent key={"faq_" + i} question={faq.question} answer={faq.answer} lastIndex={i === faqs?.length - 1} />
+          <FaqComponent key={"faq_" + i} question={faq.question} answer={faq.answer} type ={type} actions={faq.answer} content ={faq.content}lastIndex={i === faqs?.length - 1} />
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)